Stockwatch/Stockwatch.Model/Stockdata.cs

```diff
@@ -1,12 +1,12 @@
 ﻿namespace Stockwatch.Model
 {
-    public class Stockdata
+    public class StockData
```

Comment:
File name remains the same

Reply:
As i mentioned in the last PR comments - StockAlertRange would be a better name

Reply:
Yes this has been changed to "StockAlertRange"